app/components/SongCard/tests/index.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ *
+ * Tests for MusicCard
+ *
+ */
+
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom'
+import { renderWithIntl } from '@utils/testUtils';
+import SongCard from '../index';
+
+describe('<SongCard/> tests', () => {
+  const song = {
+    shortDescription: 'Song Description',
+    artworkUrl100:
+      'https://www.google.com/url?sa=i&url=https%3A%2F%2Fdribbble.com%2Fshots%2F14395014-Music-Logo&psig=AOvVaw3I_T0J8_ZRKW_g4VF_KmKz&ust=1630933127025000&source=images&cd=vfe&ved=2ahUKEwiT5uO-8efyAhWVHLcAHVSbCvQQjRx6BAgAEAk',
+    artistName: 'Artist Name',
+    previewUrl: 'https://mockurl.com'
+  };
+
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderWithIntl(<SongCard song={song} />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should contain 1 SongCard component', () => {
```

Comment:
add more tests to check if the internals of the component are getting rendered

Reply:
Added more tests accordingly

Reply:
please also check if the content you are trying to render is rendered correctly